airflow/providers/microsoft/azure/triggers/data_factory.py

```diff
@@ -30,7 +31,6 @@ class ADFPipelineRunStatusSensorTrigger(BaseTrigger):
     """
     ADFPipelineRunStatusSensorTrigger is fired as deferred class with params to run the
     task in trigger worker, when ADF Pipeline is running
-
```

Comment:
There needs to be an empty between docstring description and parameter directives to make sure the params are rendered correctly in the Python API docs.